Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,20 @@
+$urlImagen: "./assets/triangle.svg";
+
+.title {
+  position: relative;
+
+  .img {
+    background-image: url($urlImagen);
+    background-repeat: no-repeat;
+    margin: 10px 0 0 10px;
+    padding: 60px;
+    position: absolute;
+  }
+
+  .text {
+    font-family: "Open Sans", sans-serif;
+    font-size: 25px;
+    margin: 60px 0 0 80px;
+    position: absolute;
```

Comment:
don't use position absolute for both elements, just the image should be absolute positioned, the text should use its space